fixtures/fastlane/fastlane/Appfile

```diff
@@ -0,0 +1,3 @@
+app_identifier("org.aerogear.integrationtests")
+apple_id("jhellar@redhat.com")
```

Comment:
This needs to be apple developer account. I don't think we have any shared one.

Reply:
We should then create one, because if there is a problem with the account the only way is to ask you, and I wouldn't like to disturb you if for example, you are in PTO

Reply:
I've created a task for that: https://issues.jboss.org/browse/AGMS-492